lib/config.js

```diff
@@ -61,6 +62,16 @@ function loadConfig(filePath) {
         }
     }
 
+    // Recursively merge parent configurations
+    if (config.extends) {
+        var parentPath = (!isAbsolutePath(config.extends) ?
+            path.join(path.dirname(filePath), config.extends) :
+            config.extends
+        );
+
+        config = util.mergeConfigs(loadConfig(parentPath), config);
```

Comment:
Yes, adding file path to the error and then combining all of them into something meaningful would probably be a good idea. I think the message above is fine. It's descriptive enough so that you could fine and fix an issue.